wibu_catalog/templates/html/checkout.html

```diff
@@ -0,0 +1,154 @@
+            </div>
+            <div class="form-group">
+                <label for="city">{% trans "City" %} <span>*</span></label>
+                <input type="text" id="city" name="city" required>
+            </div>
+            <div class="form-group">
+                <label for="phone">{% trans "Phone Number" %} <span>*</span></label>
+                <input type="tel" id="phone" name="phone" required>
+            </div>
+            <div class="form-group">
+                <label for="email">{% trans "Email Address" %} <span>*</span></label>
+                <input type="email" id="email" name="email" required>
+            </div>
+            <div class="form-group">
+                <label for="password">{% trans "Create Account Password" %} <span>*</span></label>
+                <input type="password" id="password" name="password" required>
+            </div>
+            <a href="{% url 'order-confirmation' %}" class="order-button">{% trans "PROCEED TO CHECKOUT" %} →</a>
+            <input type="hidden" id="cart_data" name="cart_data" value=""> 
+        </form>
+    </div>
+
+    <!-- Order Summary Section -->
+    <div class="order-summary">
+        <h2>{% trans "Your Order" %}</h2>
+        <div class="cart-summary">
+            <div class="summary-item">
+                <span>{% trans "Subtotal" %}</span>
+                <span id="subtotal">₫{{ cart_total|floatformat:2 }}</span>
+            </div>
+            <div class="summary-item">
+                <span>{% trans "Total" %}</span>
+                <span id="order-total">₫{{ cart_total|floatformat:2 }}</span>
+            </div>
+        </div>
+        
+
+        <!-- Payment Methods Section -->
+        <div class="payment-method" style="display: none;">
+            <input type="radio" id="bank-transfer" name="payment-method" onclick="showDetails('bank-details')">
+            <label for="bank-transfer">{% trans "Bank Transfer" %}</label>
+        </div>
+        
+        <div id="bank-details" class="payment-details" style="display:none;">
+            <p><strong>{% trans "Please transfer the payment to the following account:" %}</strong></p>
+            <hr>
+            <p><strong>VIETCOMBANK</strong></p>
+            <p><strong>{% trans "Instructions:" %}</strong></p>
+            <p>&#9888; {% trans "Your Name + Phone Number + Item Code" %}</p>
+            <hr>
+            <p>{% trans "Once payment is made, notify the shop owner by texting the following format to 0938 100 525:" %}</p>
+            <p>&#9888; {% trans "Your Name + Phone Number + Item Code" %}</p>
+            <p>{% trans "Example: Nguyen Van A, 0909 909 090, PK026" %}</p> 
+        </div>
+
+        <div class="payment-method" style="display:none;" >
+            <input type="radio" id="momo-payment" name="payment-method" onclick="showDetails('momo-payment-details')">
+            <label for="momo-payment">{% trans "Payment via MoMo" %}</label>
+        </div>
```

Comment:
phần thanh toán với bên thứ 3 này có triển khai đc ko e? nếu ko thì em ẩn trên giao diện đi nhé
chỉ giữ phương thức COD là đc rồi

Reply:
em ẩn rồi ạ

Reply:
em set none để các phương thức khác không hiển thị ra rồi ạ